.gitignore

```diff
@@ -7,6 +7,7 @@
 .gogradle/
 build/
 vendor/
+.gradletasknamecache
```

Comment:
Add this to the apache rat configuration in the root build.gradle

Reply:
ok